### PR TITLE
Make steering fault recovery much better

### PR DESCRIPTION
**Pull request by Jamezz**
* Prevent torque from being requested while we're below the SCM minimum speed. No point, and I think we piss off the OEM hardware because of it.

* Extend the fault recovery wait. Again, I think it's too aggressive. 1 second noticeably recovers quickly.

* If we find an LKAS fault now, trigger steer error (NOT SURE ABOUT THIS YET)

### CODE REVIEW

File: selfdrive/car/gm/carcontroller.py [hunk 1]
```diff
@@ -176,10 +176,17 @@ def update(self, sendcan, enabled, CS, frame, actuators, \
     if self.lkas_active and CS.lkas_status != 1:
       # SCM ignores steering command. Workaround is to
       # temporary disable steering command.
-      # Re-enabling after 200ms seems to be working great
+      # Re-enabling after 1 second works better.
       self.lkas_active = False
       if self.inhibit_steer_for == 0:
-        self.inhibit_steer_for = 20
```
Comment (geohot) [inline selfdrive/car/gm/carcontroller.py]: It should throw an alert while this is happening

File: selfdrive/manager.py [hunk 2]
```diff
@@ -479,7 +479,7 @@ def manager_thread():
     should_start = should_start and avail > 0.02
 
     # require usb power
-    should_start = should_start and msg.thermal.usbOnline
+    #should_start = should_start and msg.thermal.usbOnline #Test fixing OP3T bug.
```
Comment (geohot) [inline selfdrive/manager.py]: Remove this?

Comment (Jamezz) [reply selfdrive/manager.py]: Whoops... that was clearly on the wrong branch :)

File: selfdrive/car/gm/carcontroller.py (hunk 1 above)
Comment (Jamezz) [reply selfdrive/car/gm/carcontroller.py]: Steering temp unavail is presented until recovery is completed (interface.py)

**Comment by vntarasov** [comment]
Fixed in upstream.

**Pull request closed without merging**